README.md

```diff
@@ -140,22 +145,22 @@ Deploy Azure IoT Hub and configure service communication to Azure Storage for Fi
 
 To validate a client device on public Internet can leverage file upload with a custom domain mapping, you will use two Terminal windows to run a client and a server .NET sample app.
 
-Run the File Upload notifications server app
+Run the File Upload notifications server application:
 
-* In a new terminal `CD` into the directory `./src/ServerSideFileNotification/`.
-* Build the .NET app: `dotnet build`
-* Run the sample app and leave it running. You will need to pass in the Service connection string from the `./deploy/quickstart-add-iothub.sh` script.
+1. In a new terminal `CD` into the directory `./src/ServerSideFileNotification/`.
+2. Build the .NET app: `dotnet build`.
+3. Run the sample application. You will need to pass in the Service connection string from the `./deploy/quickstart-add-iothub.sh` script.
 
 ```bash
 dotnet run "<iot hub service connection string>"
 ```
 
-* Leave this running.
+4. Leave this application running.
 
-Run the IoT Client sample app
+Run the IoT Client sample application:
 
-* In a new bash terminal `CD` into the directory `./src/SampleIoTClientFileUpload/`.
-* Prepare a `.env` file for the required variables:
+1. In a new bash terminal `CD` into the directory `./src/SampleIoTClientFileUpload/`.
+2. Prepare a `.env` file for the required variables:
```

Comment:
~~~suggestion
1. In a new bash terminal change into the IoT client directory

    ```bash
    ./src/SampleIoTClientFileUpload/
    ```

1. Prepare a `.env` file for the required variables.
~~~